cli/internal/fs/turbo_json_test.go

```diff
@@ -245,6 +246,40 @@ func Test_TaskOutputsSort(t *testing.T) {
 	assert.False(t, cmp.DeepEqual(taskOutputs, sortedOutputs)().Success())
 }
 
+func Test_ReadTurboConfig_VagueTurboVersion(t *testing.T) {
+	testDir := getTestDir(t, "vague_version")
+
+	packageJSONPath := testDir.UntypedJoin("package.json")
+	rootPackageJSON, pkgJSONReadErr := ReadPackageJSON(packageJSONPath)
+
+	if pkgJSONReadErr != nil {
+		t.Fatalf("invalid parse: %#v", pkgJSONReadErr)
+	}
+
+	_, turboJSONReadErr := ReadTurboConfig(testDir, rootPackageJSON)
+
+	expectedErrorMsg := "turbo.json: invalid version listed in turboVersion, only exact versions are allowed: Invalid Semantic Version"
+
+	assert.EqualErrorf(t, turboJSONReadErr, expectedErrorMsg, "Error should be: %v, got: %v", expectedErrorMsg, turboJSONReadErr)
+}
+
+func Test_ReadTurboConfig_DoubleTurboVersion(t *testing.T) {
+	testDir := getTestDir(t, "double_version")
+
+	packageJSONPath := testDir.UntypedJoin("package.json")
+	rootPackageJSON, pkgJSONReadErr := ReadPackageJSON(packageJSONPath)
+
+	if pkgJSONReadErr != nil {
+		t.Fatalf("invalid parse: %#v", pkgJSONReadErr)
+	}
+
+	_, turboJSONReadErr := ReadTurboConfig(testDir, rootPackageJSON)
+
+	expectedErrorMsg := "Found versions for turbo in both turbo.json and root package.json. Please delete the package.json dependency"
```

Comment:
> Please delete the package.json dependency

Is it possible to make this more actionable like: "Please run `npm uninstall -D turbo`" (except `npm` is the detected package manager) and `-D` is based on whether it's a dependency or devDependency).

If package manager/copy-pastable command is too far of a stretch, maybe just:

> "Please remove "turbo" from `dependencies`"

or

> "Please remove "turbo" from `devDependencies`"